hibernate-core/src/test/java/org/hibernate/orm/test/sql/exec/SmokeTests.java

```diff
@@ -325,21 +323,14 @@ public void testQueryConcurrency(SessionFactoryScope scope) throws InterruptedEx
 
 		final ExecutorService executor = Executors.newFixedThreadPool( 5 );
 
-		try {
-			for ( int f = 0; f < numberOfForks; f++ ) {
-				final ArrayList<Callable<String>> tasks = CollectionHelper.arrayList( numberOfIterations );
+		for ( int f = 0; f < numberOfForks; f++ ) {
+			final ArrayList<Callable<String>> tasks = CollectionHelper.arrayList( numberOfIterations );
 
-				for ( int i = 0; i < numberOfIterations; i++ ) {
-					tasks.add( () -> executeQueriesForConcurrency( scope ) );
-				}
-
-				executor.invokeAll( tasks );
```

Comment:
The above statement would wait patiently for the tasks to finish. The timeout config statement later won't take effect for it has no chance to be run.

Reply:
Good catch!